src/pages/[platform]/build-a-backend/data/data-modeling/relationships/index.mdx

```diff
@@ -62,17 +62,15 @@ const schema = a.schema({
     teamId: a.id(),
     // 2. Create a belongsTo relationship with the reference field
     team: a.belongsTo('Team', 'teamId'),
-  })
-  .authorization(allow => [allow.publicApiKey()]),
+  }),
 
   Team: a.model({
     mantra: a.string().required(),
     // 3. Create a hasMany relationship with the reference field
     //    from the `Member`s model.
     members: a.hasMany('Member', 'teamId'),
-  })
-  .authorization(allow => [allow.publicApiKey()]),
-});
+  }),
+}).authorization(allow => [allow.publicApiKey()]);
```

Comment:
Let's drop the array since it's just a single rule here (and below)

Reply:
Ah, touche. Didn't even notice I'd arraytized it. Fixing ...

Reply:
Done. Made parenthesis consistent here as well.